pages/file-input/simple.page.tsx

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+import React, { useState } from 'react';
+
+import { Box, ColumnLayout, FileInput } from '~components';
+
+export default function DateInputScenario() {
+  const [files, setFiles] = useState<File[]>([]);
+  const [files2, setFiles2] = useState<File[]>([]);
+
+  return (
```

Comment:
This page and the one for dropzone do no include screenshot area. I recommend we add it at least to the dropzone and use for screenshot tests.